src/npm/describeNpmTag.ts

```diff
@@ -0,0 +1,9 @@
+import spawn from "../utils/spawn.js";
+import appendCmdIfWindows from "./utils/appendCmdIfWindows.js";
+import npmLogger from "./utils/npmLogger.js";
+
+export default async function (packageName: string, tag = "latest") {
+  const params = ["show", [packageName, tag].join("@"), "version"];
```

Comment:
Where is the documentation for `npm show`? I can't seem to find it here https://docs.npmjs.com/cli/v9/commands

Reply:
https://docs.npmjs.com/cli/v9/commands/npm-view
Tho I'll rename to view.